requirements/requirements.txt

```diff
@@ -2,6 +2,6 @@ pandas
 numpy
 requests>2,<3
 urllib3>1.24,<2
-websocket-client>=0.56.0,<1
-websockets>=8.0,<9
+websocket-client>=0.56.0,<2
+websockets>=8.0,<10
```

Comment:
In https://github.com/alpacahq/alpaca-trade-api-python/pull/447 it says:
pyup.io/safety-ci — websockets 8.1 has known security vulnerabilities.

Shouldn't we go with a higher minimum version because of this?

Reply:
if we do, we will prevent users that may use other packages that require that version (8.1 was the default version for a very long time).
We could recommend using a higher version, but I don't think we should prevent it

Reply:
added this note and merging: 
![image](https://user-images.githubusercontent.com/78740792/122514530-c2bcab80-d014-11eb-8272-1805c2645904.png)